CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 # This file is part of Brayns <https://github.com/BlueBrain/Brayns>
 
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
-project(Brayns VERSION 0.7.1)
+project(Brayns VERSION 0.8.0)
```

Comment:
I'm not super happy with that bump, given that a patch release is kind of mandatory after the fix I did yesterday (and maybe one more needed, depending if @favreau finds the material problem).

Reply:
What should I do about that?